editor/src/com/talosvfx/talos/editor/GridRendererWrapper.java

```diff
@@ -38,4 +47,16 @@ public void draw (Batch batch, float parentAlpha) {
         gridRenderer.drawGrid(batch, shapeRenderer);
         batch.begin();
     }
+
+    protected float pixelToWorld (float pixelSize) {
+        tmp.set(0, 0, 0);
+        camera.unproject(tmp);
+        float baseline = tmp.x;
+
+        tmp.set(pixelSize, 0, 0);
+        camera.unproject(tmp);
+        float pos = tmp.x;
+
+        return Math.abs(pos - baseline);
+    }
```

Comment:
I don't mind duplicating because this class is going to be removed once we switch everything to viewport class extensions